css/style.css

```diff
@@ -526,3 +548,95 @@
   order: 0;
   flex-grow: 0;
 }
+
+/*  Contact form style */
+
+.contact-wrapper {
+  margin-top: 50px;
+  background-image: url("../Image/contact-bg.svg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  padding-top: 50px;
+  padding-bottom: 45px;
+}
+
+.contact-container {
+  display: flex;
+  flex-direction: column;
+  background-image: url("../Image/contact-form-shapes-mobile.png");
+  margin-top: 103px;
+  background-size: 65% 99%;
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-position-y: 15px;
+  z-index: -1;
+}
+
+.contact-container h1 {
+  font-family: Poppins, 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  color: #fff;
+  padding-top: 103px;
+  text-align: center;
+}
+
+.my-form {
+  display: flex;
+  flex-direction: column;
+  margin-top: 20px;
+}
+
+.my-form input {
+  height: 48px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  padding: 15px 38px 15px 16px;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  border-radius: 8px;
+  margin: 0 24px 24px;
+}
+
+.my-form textarea {
+  height: 170px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  padding: 15px 38px 15px 16px;
+  border-radius: 8px;
+  margin: 0 24px 24px;
+}
+
+.my-form button:hover {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 12px;
+  width: 150px;
+  height: 48px;
+  background-color: #2230d2;
```

Comment:
[OPTIONAL]

It seems like your button wiggles when hovering and doesn't have the right color, the hex code should be `#6070FF`, make sure you're using the same margin/padding/border! :sweat_smile: